housekeeping/azure-pipelines.yml

```diff
@@ -230,6 +232,26 @@ jobs:
       inputs:
         helmVersionToInstall: 'latest'
 
+    - task: AzureCLI@2
+      name: aksFWCheck
+      displayName: AKS FW status checker Master
+      inputs:
+        azureSubscription: 'NHSAPP-BuyingCatalogue (Non-Prod)'
+        scriptType: 'bash'
+        scriptLocation: 'inlineScript'
+        inlineScript: |        
+            FWStatus=$(az aks show -n gpitfutures-development-aks -g gpitfutures-development-rg-aks --query apiServerAccessProfile.authorizedIpRanges)
+            if [[ -n "$FWStatus" ]]; then
+            echo "AKS FW is on"; else echo "AKS FW is off"; fi
+
+            while [ -n "$FWStatus" ]
+            do
+            echo "disabling AKS Firewall";
+            az aks update -n gpitfutures-development-aks -g gpitfutures-development-rg-aks --api-server-authorized-ip-ranges "" | grep authorizedIpRanges;
+            az aks wait -n gpitfutures-development-aks -g gpitfutures-development-rg-aks --updated;
+            break;
+            done
```

Comment:
Is the `while` loop needed here? Won't there only ever be a single iteration due to the `break` statement on line 252?

Reply:
Yep